admin/src/scenes/volontaires/view/applicationList2.js

```diff
@@ -0,0 +1,377 @@
+                      "Département de la mission": data.missionDepartment,
+                      "Région de la mission": data.missionRegion,
+                      "Candidature créée lé": data.createdAt,
+                      "Candidature mise à jour le": data.updatedAt,
+                      "Statut de la candidature": translate(data.status),
+                      Tuteur: data.tutorName,
+                      "Pièces jointes à l’engagement": translate(`${optionsType.reduce((sum, option) => sum + data[option].length, 0) !== 0}`),
+                    };
+                  });
+                }}
+              />
+            </div>
+          </ReactiveBase>
+        </div>
+      </div>
+
+      {/* Mission card */}
+      <div className="mx-14 ">
+        {applications.map((hit, i) => (
+          <Hit key={hit._id} young={young} hit={hit} index={i} onChangeApplication={onChangeApplication} optionsType={optionsType} />
+        ))}
+
+        {applications.length ? null : <NoResult>Aucune candidature n&apos;est liée à ce volontaire.</NoResult>}
+      </div>
+    </div>
+  );
+}
+
+const Hit = ({ hit, index, young, onChangeApplication, optionsType }) => {
+  const [mission, setMission] = useState();
+  const [modalDurationOpen, setModalDurationOpen] = useState(false);
+  const [modal, setModal] = useState({ isOpen: false, onConfirm: null });
+  const [tags, setTags] = useState();
+
+  const history = useHistory();
+  useEffect(() => {
+    (async () => {
+      if (!hit.missionId) return;
+      const { ok, data, code } = await api.get(`/mission/${hit.missionId}`);
+      if (!ok) {
+        capture(code);
+        return toastr.error("Oups, une erreur est survenue", code);
+      }
+      setMission(data);
+      const t = [];
+      data?.city && t.push(data?.city + (data?.zip ? ` - ${data?.zip}` : ""));
+      (data?.domains || []).forEach((d) => t.push(translate(d)));
+      setTags(t);
+    })();
+  }, []);
+
+  // const theme = {
+  //   background: {
+  //     WAITING_VALIDATION: "bg-sky-100",
+  //     WAITING_VERIFICATION: "bg-sky-100",
+  //     WAITING_ACCEPTATION: "bg-orange-500",
+  //     VALIDATED: "bg-[#71C784]",
+  //     DONE: "bg-[#5694CD]",
+  //     REFUSED: "bg-[#0B508F]",
+  //     CANCEL: "bg-[#F4F4F4]",
+  //     IN_PROGRESS: "bg-indigo-600",
+  //     ABANDON: "bg-gray-50",
+  //   },
+  //   text: {
+  //     WAITING_VALIDATION: "text-sky-600",
+  //     WAITING_VERIFICATION: "text-sky-600",
+  //     WAITING_ACCEPTATION: "text-white",
+  //     VALIDATED: "text-white",
+  //     DONE: "text-white",
+  //     REFUSED: "text-white",
+  //     CANCEL: "text-[#6B6B6B]",
+  //     IN_PROGRESS: "text-white",
+  //     ABANDON: "text-gray-400",
+  //   },
+  // };
+
+  const [openModalPJ, setOpenModalPJ] = useState(false);
+
+  if (!mission) return null;
+  return (
+    <div className="relative  w-full  bg-white shadow-nina rounded-xl p-4 border-[1px] border-[#ffffff] hover:border-gray-200 drop-shadow-md mb-4">
```

Comment:
```suggestion
    <div className="relative  w-full  bg-white shadow-nina rounded-xl p-4 border-[1px] border-white hover:border-gray-200 drop-shadow-md mb-4">
```
plus simple je pense !?